.travis.yml

```diff
@@ -4,12 +4,11 @@ python:
   - 3.6
   - 3.5
   - 3.4
-  - 3.3
```

Comment:
please also remove 2.6 and 3.3 from the `Makefile`, `setup.py` classifiers, `tox.ini`, `.python-version` and any docs where mentioned (`installation.rst` and `testing.rst` is where I also see it)